spec/requests/api/requests_spec.rb

```diff
@@ -248,6 +248,31 @@
       expect(response.parsed_body).to match(expected)
       expect(response).to have_http_status(:ok)
     end
+
+    it "exposes allowed_tags in the request resources" do
```

Comment:
So what you're saying is, instead of writing a test on the API side we should write it only on the model side? ...which kinda makes sense in this situation. 
Although the only reason I added a virtual_column was because I have this requirement to access it via the API. Without this requirement in the picture, the model does not need any changes.

How about we include both tests?